src/nbl/video/IAPIConnection.cpp

```diff
@@ -4,13 +4,17 @@
 #include "nbl/video/utilities/renderdoc.h"
 #include "nbl/video/utilities/ngfx.h"
 
+// TODO: temporary hopefully
+#include "C:\Program Files\NVIDIA Corporation\Nsight Graphics 2024.1.0\SDKs\NsightGraphicsSDK\0.8.0\include\NGFX_Injection.h"
+
```

Comment:
@AnastaZIuk we need your help for NGFX SDK detection with Cmake and NBL_BUILD_WITH_NGFX or NBL_NGFX_FOUND

Reply:
done as part of https://github.com/Devsh-Graphics-Programming/Nabla/pull/789